project/Dependencies.scala

```diff
@@ -541,16 +541,14 @@ object Dependencies {
   val `persistence-jdbc-core` = libraryDependencies ++= Seq(
     slf4jApi,
     akkaPersistenceJdbc,
-    playJdbc
-  )
+    playJdbc,
 
-  val `persistence-jdbc-javadsl` = libraryDependencies ++= Seq(
     h2 % Test
```

Comment:
I think explicit external-dependencies are good.

Reply:
Sorry, I don't totally understand this comment. Are you saying that the change is good, or that the change is not explicit enough?

Reply:
Sorry, I had a companion comment to this one which added context and I just realised I ended up not including that comment on the review.

What I tried to say is that I don't like depending on transitive dependencies. If `persistence-jdbc-javadsl` needs `logback` then I prefer if it's there explicitly. At same time, I won't go bananas reviewing that every project declares all its dependencies explicitly.

So in the PR I'd keep `logback` in all downstream projects that already had it.

Apologies for the confusion.

Reply:
OK I will change that, but I'll also note that none of these projects have any explicit dependency on logback. It's just that we want to bind a `slf4j` implementation when running tests so that we can see log output. Otherwise you get a warning and an implicit binding to `slf4j-nop` (which discards all log messages).

Still, I agree it's better to be explicit, and I kind of think it's a smell for projects to depend on another project's test classpath anyway (though we're doing it a lot in Lagom).

Reply:
Sorry... was looking at the wrong dependency change, but it's a similar situation. The dependency on H2 is already implicit.

Reply:
> Still, I agree it's better to be explicit, and I kind of think it's a smell for projects to depend on another project's test classpath anyway (though we're doing it a lot in Lagom).

Couldn't agree more.